### PR TITLE
Code clean up

### PR DESCRIPTION
**Pull request by GucciBanana11**
Cleaned up code, combining `main`, `shooter tuning stuff`, and other various insignificant branches. Branches will be deleted by me once this is merged.

### CODE REVIEW

**Review by Stubbynarwhal — APPROVED** [review]
Kind of awesome 
👍🤩

**Comment by awfulcarpet** [comment]
just a note but, specify which branches are going to be deleted before deleting them

File: src/main/java/frc/robot/commands/ShootCommand.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
 public class ShootCommand extends Command {
   private final Shooter shooter;
   private final Intake intake;
-  private double speed = 0.2; // TODO Change to intake motor speed (in rotations per second)
+  private double speed = 0.2; // Intake motor speed (in percentage of power)
```
Comment (awfulcarpet) [inline src/main/java/frc/robot/commands/ShootCommand.java:14]: we should probably keep the TODO

**Review by awfulcarpet — APPROVED** [review]

**Comment by awfulcarpet** [comment]
don't merge until it is tested to work

**Review by centiwati — APPROVED** [review]

**Pull request merged**